src/AdminTab/TableList.tsx

```diff
@@ -39,10 +45,26 @@ export const TableList: FunctionComponent = () => {
         <Column
           key="actions"
           align="right"
-          render={(_: unknown, table: ImportedTable) => (
-            <Button type="text" shape="circle" onClick={() => removeTable(table.name)}>
-              <DeleteOutlined />
-            </Button>
+          render={(_: unknown, table: ImportedTable, index: number) => (
+            <>
+              <Tooltip
+                placement="left"
+                visible={showMetabaseHint && !metabaseHintHovered && index == 0}
```

Comment:
`index === 0`. We should make a linter warning because `==` is almost never used.

Reply:
thx, I keep missing the extra =, sigh

Reply:
Can you look into enabling the relevant linter rule?